pom.xml

```diff
@@ -116,48 +157,57 @@
             <artifactId>mapstruct</artifactId>
             <version>${org.mapstruct.version}</version>
         </dependency>
-        <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-engine</artifactId>
-            <scope>test</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>org.mockito</groupId>
-            <artifactId>mockito-junit-jupiter</artifactId>
-            <scope>test</scope>
-        </dependency>
-
-
     </dependencies>
 
+    <repositories>
```

Comment:
I know it was me who suggested you put this in, but I have since learnt that this is in the ch-parent-pom so I was wrong, it is not needed. Other than that I am happy to approve.